src/segmantic/seg/monai_unet.py

```diff
@@ -401,7 +514,7 @@ def predict(
             settings = json.load(json_file)
         net = Net.load_from_checkpoint(f"{model_file}", **settings)
     else:
-        net = Net.load_from_checkpoint(f"{model_file}")
+        net = Net.load_from_checkpoint(f"{model_file}", channels=channels, strides=strides, dropout=dropout)
```

Comment:
shouldn't be necessary if you trained with `self.save_hyperparameters()`. Are you running this function using checkpoints trained before this PR and need it for backwards compatibility?

Reply:
Yes, I was a bit unsure about this one. I remembered you said that you added all these things to be saved in the .ckpt file, but exactly so I would use an older model it would likely complain right? So yes, I added it with backward compatibility in mind. Should that be done in a different way then maybe?